RecoLocalMuon/GEMRecHit/python/gemRecHits_cfi.py

```diff
@@ -14,5 +14,5 @@
 from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
 from Configuration.Eras.Modifier_phase2_GEM_cff import phase2_GEM
 
-run3_GEM.toModify(gemRecHits, ge21Off=True)
+run3_GEM.toModify(gemRecHits, ge21Off=True, applyMasking=True)
```

Comment:
@yeckang, let me ask the question I asked in https://github.com/cms-sw/cmssw/pull/44085#discussion_r1544609272.

Afaik, the corresponding module in the Run-3 HLT menu is as [1] ([link](https://raw.githubusercontent.com/cms-sw/cmssw/405f58489884b0ff1dbd1ffd6cf0343ef1db407c/HLTrigger/Configuration/test/OnLine_HLT_GRun.py)).

Does the change in this PR need to be applied to the Run-3 HLT menu ? (cc: @cms-sw/hlt-l2)

[1]
```py
process.hltGemRecHits = cms.EDProducer( "GEMRecHitProducer",
    recAlgoConfig = cms.PSet(  ),
    recAlgo = cms.string( "GEMRecHitStandardAlgo" ),
    gemDigiLabel = cms.InputTag( "hltMuonGEMDigis" ),
    applyMasking = cms.bool( False ),
    ge21Off = cms.bool( False )
)
```

Reply:
@missirol In my opinion, it is also good to apply there. The update would merge the fragmented rechits, which might be a single hit originally.

Reply:
@yeckang can you (or someone else from GEM DPG) open please a [CMSHLT](https://its.cern.ch/jira/projects/CMSHLT) to propose the change in the menu ?

Reply:
I took the liberty to go ahead and open the ticket, [CMSHLT-3420](https://its.cern.ch/jira/browse/CMSHLT-3420).

Reply:
Thanks Marino, though I have to admit that I find the fact that TSG coordination needs to babysit DPGs to make sure they port offline reconstruction developments to online as well somewhat distressful.